### ISSUE TITLE
[HTML] Move embedded language scopes to inside tags

### ISSUE THREAD

**Issue by groteworld** [sublimehq/Packages#106]
In the HTML syntax currently the scopes `source.js.embedded.html` and `source.css.embedded.html` are being placed on the `<script>` and `<style>` tags and their contents respectively. These scopes, in reality, should be placed *within* the tags, as this is the embedded JavaScript and CSS.

Thanks for your help and time!

**Comment by grotewold** [comment]
Any update on this, @sublimehq?

**Comment by kaste** [comment]
@grotewold  Do you have a patch?

**Comment by wbond** [comment]
This should be resolved by 0417e42da416ec7f4fced63e84316944fc5d257f

**Comment by groteworld** [comment]
The new employee is god-like indeed! As always your contributions are invaluable. Thank you again! :beers:

**Comment by groteworld** [comment]
CSS scope seem to be still grabbing the end of the of the `<style>` tag. See example below:

```html
<html>
    <style>
        .test
    </style>
</html>
```

ST console
```python
>>> my_view = sublime.active_window().views()[0]
>>> my_view.substr(my_view.text_point(1, 6))
'>'
>>> my_view.scope_name(my_view.text_point(1, 6))
'text.html.basic source.css.embedded.html punctuation.definition.tag.end.html '
```

**Comment by wbond** [comment]
Thanks for the bug report. Should be fixed in f9b9e10b8d38b233c1078a425b6c1c026a62aa63.
